Lib/test/test_xmlrpc.py

```diff
@@ -343,6 +343,24 @@ def run_server():
             self.assertEqual(p.method(), 5)
             self.assertEqual(p.method(), 5)
 
+    def test_simple_xml_rpc_dispatcher_using_dispatch_func(self):
```

Comment:
The test is passed when leave `Lib/xmlrpc/server.py` unchanged. Thus it doesn't test changes.